README.md

```diff
@@ -0,0 +1,47 @@
+# Pulumi Workshops
+
+This is the official repo for public Pulumi workshops. If a workshop is running from Pulumi, you'll likely find the code here!
+
+Each workshop has a directory. Inside the directory is a README with details about the workshop and what to expect and subdirectories with different labs from the workshop. You generally are not expected to checkout a repo to do a workshop; code provided is for copying and pasting into your own workspace.
+
+Note that some of the workshops on this list may be identical to others. Use workshops marked `ACTIVE` only! If you're updating a workshop, please update the metadata, this table, and move it out of the `archive` directory so we know it's currently in use.
+
+Status | Topic | Language | Provider | Link
```

Comment:
`tree -d -L 2` will basically give you this list if you want to do away with the manual maintenance, since you've already organized things well by directory/file:

```bash
$ tree -d -L 2
.
├── archive
│   ├── aws-intro
│   ├── aws-workshop
│   ├── azure-intro
│   ├── building-a-kubernetes-platform-in-amazon-eks
│   ├── cicd-with-gha-and-pulumi-operator
│   ├── deploying-argocd-to-amazon-eks
│   ├── deploying-containers-to-ecs
│   ├── gitops-for-socks-shop
│   ├── iac-intro
│   ├── introduction-to-pulumi
│   ├── lambda-for-serverless-application-patterns
│   ├── modern-infrastructure-as-code-ts
│   ├── provision-ec2-virtual-machines
│   ├── pulumi-automationapi-workshop
│   ├── pulumi-azurenative-workshop
│   ├── pulumi-component-workshop
│   ├── pulumi-eksplatform-workshop
│   ├── pulumi-gcp-workshop
│   ├── pulumi-on-aws
│   ├── pulumi-refactoring-workshop
│   ├── qcon-workshop
│   └── testing-your-infrastructure
├── building-reusable-abstractions
│   ├── 1-functions
│   └── 2-component-resources
├── cloud-engineering-with-aws-X
│   ├── java
│   └── python
├── embedding-pulumi
│   ├── 1-getting-started
│   ├── 2-automating-commands
│   ├── 3-building-an-api
│   ├── 4-logging-and-exceptions
│   └── 5-putting-it-together
└── fundamentals
    ├── python
    └── yaml
```